MathlibTest/superscript.lean

```diff
@@ -81,3 +81,53 @@ run_cmd Lean.Elab.Command.liftTermElabM do
   Lean.logInfo <| ← Lean.PrettyPrinter.ppTerm <| ← `(term| ($a)$α:subscript)
 
 end
+
+section delab
+
+open Lean PrettyPrinter.Delaborator SubExpr
+open Mathlib.Tactic (delabSubscript delabSuperscript)
+
+private def check_subscript {α : Type} (_ : α) := ()
+local syntax:arg "#" noWs subscript(term) : term
```

Comment:
Let's pick something more obvious, like
```suggestion
local syntax:arg "subscript_test(" noWs subscript(term) noWs ")" : term
```

Reply:
I think I have implemented this change, but I haven't quite understood what you mean by trying to subscript a superscript (or how best to test that).